modules/schat.cpp

```diff
@@ -226,9 +230,15 @@ class CSChat : public CModule
 					Table.SetCell("Host", pSock->GetRemoteIP());
 					Table.SetCell("Port", CString(pSock->GetRemotePort()));
 					SSL_SESSION *pSession = pSock->GetSSLSession();
+#ifdef HAVE_OPAQUE_SSL_SESSION
+					if (pSession)
+						Table.SetCell("Cipher", SSL_CIPHER_get_name(SSL_SESSION_get0_cipher(pSession)));
+#else
 					if (pSession && pSession->cipher && pSession->cipher->name)
 						Table.SetCell("Cipher", pSession->cipher->name);
-
+#endif
+					else
+						Table.SetCell("Cipher", "None");
```

Comment:
I looked into it a bit, `SSL_CIPHER_get_name` seems to have been available in 0.9.8 (does znc still support older than that?) - what's changed is that 1.1 now requires it to extract the name from the opaque cipher.

It's `SSL_SESSION_get0_cipher` that was only implemented a few weeks before 1.1's release and is needed to get the cipher out of the opaque session, so this is what I'd implement a shim for (unless pre-0.9.8 support is also still needed).